mkosi.presets/10-initrd/mkosi.conf

```diff
@@ -5,11 +5,9 @@ Bootable=auto,yes
 
 [Output]
 Format=cpio
-# TODO: Switch to zstd once we stop building CentOS Stream 8.
-CompressOutput=xz
 
 [Content]
-BaseTrees=mkosi.output/base
+BaseTrees=../../mkosi.output/base
```

Comment:
what's this change for?

Reply:
10-initrd.conf moved to 10-initrd/mkosi.conf. When parsing mkosi.conf.d or mkosi.presets, relative paths in files are interpreted relative to the parent directory, relative paths in mkosi.conf in directories are interpreted relative to that directory. So since we went from a file to a directory, we have to adjust the relative path to match.

Reply:
This will get a bit confusing, it will need some good documentation